Solutions/Network Session Essentials/Hunting Queries/Detect Outbound LDAP Traffic(ASIM Network Session schema).txt

```diff
@@ -0,0 +1,20 @@
+id: 5dca6047-24ed-4eb7-b44e-ec7f1bf42621
+name: Detect Outbound LDAP Traffic(ASIM Network Session schema)
+description: |
+  'Malicious actors often abuse misconfigured LDAP servers or applications that use the LDAP servers in organizations. Outbound LDAP traffic should not be allowed outbound through your perimeter firewall. This search will help determine if you have any LDAP connections to IP addresses outside of private (RFC1918) address space.'
```

Comment:
Hello @praveenthepro, shorten the description below 255 characters.

Reply:
reduced the  description below 255